pkgs/build-support/rust/hooks/default.nix

```diff
@@ -115,7 +115,7 @@
             "linker" = "${stdenv.cc}/bin/${stdenv.cc.targetPrefix}cc"
             "rustflags" = [ "-C", "target-feature=${
               if pkgsTargetTarget.stdenv.targetPlatform.isStatic then "+" else "-"
-            }crt-static" ]
+            }crt-static", "--remap-path-prefix", "@BUILD_TOP@=/build" ]
```

Comment:
I know I'm suggesting this late, but it wouldn't be possible to just make this relative, would it? Might make using gdb easier.

Reply:
I think I meant relative to crate roots, but in retrospect, that would make it difficult to distinguish between all the src/lib.rs files.  One day I dream of us having a "source" output we could point debug info into.